src/components/filters/RangeFilter/index.jsx

```diff
@@ -8,12 +8,23 @@ class RangeFilter extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      lowerBound: props.lowerBound ? props.lowerBound : props.min,
-      upperBound: props.upperBound ? props.upperBound : props.max,
+      lowerBound: props.lowerBound,
+      upperBound: props.upperBound,
       isDragging: false,
     };
   }
 
+  // When component receives new props, update state.lowerBound and state.upperBound
+  static getDerivedStateFromProps(props, prevState) {
```

Comment:
This is causing some problematic behavior for ranged filter tests in storybook. Because if the ranged filter don't have parent to pass down props, it won't update the `props.lowerBound` and `props.upperBound` by itself hence this is always resetting the states to undefined`

Reply:
Yes, that makes sense 🤦‍♂I actually fixed this in the slider input branch (I reverted the getDerivedState approach and used the `key` approach instead). I'll see if I can pull my changes into this branch

Reply:
Just pulled in the changes from the slider input branch -- I ran through the tests and did manual tests in storybook and the explorer and it looks good to me. Thanks so much for catching this!